src/main/java/org/fundacionjala/coding/abel/SrotTheInnerCtonnetInDsnnieedcgOredr.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.abel;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+/**
+ *
+ */
+public class SrotTheInnerCtonnetInDsnnieedcgOredr {
+
+    /**
+     *
+     */
+    private static final int MIN = 4;
+
+
+    /**
+     * @param sentence input string
+     * @return value of return
+     */
+    public String sortTheInnerContent(final String sentence) {
```

Comment:
This may be refactored like this.
```java
    String sortTheInnerContent(final String sentence) {
        String[] string = sentence.split(" ");
        for (int i = 0; i < string.length; i++) {
            string[i] = string[i].length() >= MIN ? invertirOrdenCaracteres(string[i]) : string[i];
        }
        return String.join(" ", string);
    }

    /**
     * @param word text input
     * @return value of return
     */
    private String invertirOrdenCaracteres(final String word) {
        String ini = String.valueOf(word.charAt(0));
        String fin = String.valueOf(word.charAt(word.length() - 1));
        String subWord = word.substring(1, word.length() - 1);
        String[] arrayChars = subWord.split("");
        Arrays.sort(arrayChars, Collections.reverseOrder());
        return String.join("", ini, String.join("", arrayChars), fin);
    }
```